src/model.js

```diff
@@ -119,23 +117,24 @@ export default class Model {
         /*
          Для массивов объектов из нового и старого файлов моделей
          вызываем метод _generateUrlPageMap, который строит из данных массивов
-         объекты в которых ключами являются url страниц, а значениями сами объекты
+         объекты, в которых ключами являются url страниц, а значениями сами объекты
          */
-        const newModel = this.constructor._generateUrlPageMap(currentModel);
-        const oldModel = this.constructor._generateUrlPageMap(previousModel);
+        const generateUrlPageMap = this.constructor._generateUrlPageMap;
+        const newModel = generateUrlPageMap(currentModel);
+        const oldModel = generateUrlPageMap(previousModel);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).